airflow/ui/src/pages/Run/TaskInstances.tsx

```diff
@@ -96,13 +158,53 @@ export const TaskInstances = () => {
       limit: pagination.pageSize,
       offset: pagination.pageIndex * pagination.pageSize,
       orderBy,
+      state: filteredState === null ? undefined : [filteredState],
```

Comment:
Since `state` accepts an array. We should make it a MultiSelect component.

Reply:
Thanks, done. I am appending to query parameter `state` on multiple selection so that it's sent as array to backend for filtering